arwen/contexts/metering_test.go

```diff
@@ -279,80 +282,212 @@ func TestMeteringContext_GasUsed_NoStacking(t *testing.T) {
 	require.Equal(t, uint64(1401), gasUsedByContract)
 }
 
-func TestMeteringContext_GasUsed_StackOneLevel(t *testing.T) {
-	t.Skip()
+func setUpStackOneLevel(t *testing.T) (*meteringContext, *outputContext, *contextmock.RuntimeContextMock, *vmcommon.ContractCallInput, uint64, *vmcommon.ContractCallInput) {
 	t.Parallel()
 
 	mockRuntime := &contextmock.RuntimeContextMock{}
+
 	host := &contextmock.VMHostMock{
 		RuntimeContext: mockRuntime,
 	}
 
+	output, _ := NewOutputContext(host)
+	host.OutputContext = output
+
 	contractSize := uint64(1000)
 	contract := make([]byte, contractSize)
+
 	mockRuntime.SCCodeSize = contractSize
 	mockRuntime.SCAddress = []byte("parent")
 
 	mockRuntime.SetPointsUsed(0)
-	parentInput := &vmcommon.ContractCallInput{VMInput: vmcommon.VMInput{}}
+	parentInput := &vmcommon.ContractCallInput{
```

Comment:
Consider receiving `parentInput` and `childInput` as parameters in this function, provided by whatever test calls it.